src/cluster_legacy.c

```diff
@@ -2246,9 +2257,9 @@ void clusterProcessGossipSection(clusterMsg *hdr, clusterLink *link) {
         if (node && node != myself) {
             /* We already know this node.
                Handle failure reports, only when the sender is a voting primary. */
```

Comment:
This comment needs to be updated - "only when the sender is a voting primary" is not longer accurate since recovery reports can be processed from any sender.

Reply:
yes, the comment is outdated, i updated it, thanks